src/components/blocks/Form.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+
+import { Movie } from '../../assets/js/types'
+import { genres } from '../../assets/js/utils'
+
+type Form = {
+    film: Movie,
+    setShowModal: (arg: boolean) => void
+}
+
+const Form: React.FC<Form> = ({ film, setShowModal }) => {
+    const [movie, setMovie] = useState<Movie>({...film})
+
+    const handleGenres = (event: { target: { options: HTMLOptionsCollection } }) => {
+        let options = [...event.target.options];
+        const selectedOptions = options.filter(option => option.selected)
+                                        .map((item: { value: string }) => item.value)
+        
+        setMovie({...movie, genres: selectedOptions })
+    }  
+
+    const genresSelect = genres.map((genre) => {
+        return <option key={genre} value={genre}>{genre}</option>
+    })
+
+    
+    return (
+        <div  className="modal">
+            <div className="modal-content">
+                <div className="modal-header">
+                    <span className="close" onClick={() => setShowModal(false)}>&times;</span>
```

Comment:
You just using `setShowModal(false)` multiple times. So it's better to put it in separate function:
```js
const closeShowModal = () => {
   setShowModal(false)
}
``` 

and then use it like this: `onClick={closeShowModal}`